src/main/java/com/cedrus/patpetillo/springkafkapingpong/model/PingPongBall.java

```diff
@@ -15,4 +15,9 @@
     public void returnBall() {
         this.pingPongTarget = pingPongTarget.equals(PingPongTarget.PING) ? PingPongTarget.PONG : PingPongTarget.PING;
     }
+
+    @Override
+    public String toString() {
```

Comment:
You can do this with the Lombok annotation ToString, or save yourself even more time and use the metaannotation Data